native/SampleApps/RestAPIExplorerSwift/RestAPIExplorerSwift/AppDelegate.swift

```diff
@@ -0,0 +1,264 @@
+//
```

Comment:
Could you use this copyright notice: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/libs/SalesforceAnalytics/SalesforceAnalytics/SalesforceAnalytics.h#L1

Reply:
Same created by? Or simply remove the double Copyright?